lms/envs/aws.py

```diff
@@ -896,4 +896,7 @@
 
 ############## Settings for the Enterprise App ######################
 
-ENTERPRISE_ENROLLMENT_API_URL = ENV_TOKENS.get('ENTERPRISE_ENROLLMENT_API_URL', ENTERPRISE_ENROLLMENT_API_URL)
+ENTERPRISE_ENROLLMENT_API_URL = ENV_TOKENS.get(
+    'ENTERPRISE_ENROLLMENT_API_URL',
+    (LMS_ROOT_URL or '') + '/api/enrollment/v1/'
```

Comment:
My only comment here is a take-or-leave -- you could consider something like this for readability:

```
DEFAULT_ENTERPRISE_ENROLLMENT_API_URL = (LMS_ROOT_URL or '') + '/api/enrollment/v1/'
ENTERPRISE_ENROLLMENT_API_URL = ENV_TOKENS.get(
    'ENTERPRISE_ENROLLMENT_API_URL', DEFAULT_ENTERPRISE_ENROLLMENT_API_URL
)
```